src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/service/SentenceService.kt

```diff
@@ -153,10 +153,13 @@ class SentenceService(
 
   fun getSentenceFromChargeOrUuid(chargeEntity: ChargeEntity, sentenceUuid: UUID?): SentenceEntity? = chargeEntity.getActiveSentence() ?: sentenceUuid?.let { sentenceRepository.findFirstBySentenceUuidOrderByUpdatedAtDesc(sentenceUuid) }
 
-  @Transactional(TxType.REQUIRED)
+  @Transactional(readOnly = true)
   fun findSentenceByUuid(sentenceUuid: UUID): Sentence? = sentenceRepository.findFirstBySentenceUuidOrderByUpdatedAtDesc(sentenceUuid)?.let { Sentence.from(it) }
 
-  @Transactional(TxType.REQUIRED)
+  @Transactional
```

Comment:
I'm not sure if adding `(readOnly = true)` to `@Transactional ` improves perf slightly.. one to look up, in which case may be worth adding here